sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRestrictedSearchMode.md

```diff
@@ -0,0 +1,67 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-spotenantrestrictedsearchmode
+applicable: SharePoint Online
+title: Set-SPOTenantRestrictedSearchMode
+schema: 2.0.0
+author: praporwal
+ms.author: praporwal
+ms.reviewer:
+---
+
+# Set-SPOTenantRestrictedSearchMode
+
+## SYNOPSIS
+
+Set the mode for Restricted Search setting.
+
+## SYNTAX
+
+```powershell
+Set-SPOTenantRestrictedSearchMode -Mode {Disabled | Enabled} [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Enable or disable the restricted search setting with the default being disabled. The first time when the setting is enabled the allow list is empty.
```

Comment:
> with the default being disabled [](http://example.com/codeflow?start=47&length=32)

not sure I follow this part, for the purpose of this cmdlet?

Reply:
Updated